test/src/unittests/essentia_test.py

```diff
@@ -187,6 +187,18 @@ def assertConfigureFails(self, algo, params):
         conf = lambda: algo.configure(**params)
         self.assertRaises(EssentiaException, conf)
 
+    def assertConfigureSuccess(self, algo, params):
+        try:
+            algo.configure(**params)
+        except EssentiaException:
```

Comment:
what if `configure` fails for some other exception type, not being `EssentiaException`?

Reply:
In that case, the test will return `error` instead of `fail`.
As I understand it, the goal of this assertion is to test if a particular case skips an exception that is the general rule.
The way I think it is:
pass -> ok
fail -> we caught the error but we didn't consider that this particular case should be skipped
error -> we did not catch the error

Do you think we should except any exception instead?

Reply:
Yes, I think that is fine, as the case of ERROR will help to know that we did not catch some exception inside `configure`.